pkgs/development/python-modules/face_recognition/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildPythonPackage, fetchFromGitHub, pillow, click, dlib, numpy
+, face_recognition_models, scipy, stdenv, flake8, tox
+}:
+
+buildPythonPackage rec {
+  pname = "face_recognition";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
```

Comment:
You could add a comment why you're fetching from GitHub, and maybe open an upstream issue about missing tags.

Reply:
@Ma27 Did you open an issue?